docs/netavark-firewalld-forwarding.5.md

```diff
@@ -0,0 +1,85 @@
+% NETAVARK-FIREWALLD 5 Netavark Firewalld Interactions Man Page
+% Matthew Heon
+% January 2025
+
+## Name
+
+netavark-firewalld - description of the interaction of Netavark and firewalld
+
+## Description
+
+Netavark can be used on systems with firewalld enabled without issue.
+When using the default `nftables` or `iptables` firewall drivers, on systems where firewalld is running, firewalld will automatically be configured to allow connectivity to Podman containers.
+All subnets of Podman-managed networks will be automatically added to the `trusted` zone to allow this access.
+
+### Firewalld Driver
+
+There is also a dedicated firewalld driver in Netavark.
+This driver uses the firewalld DBus API to natively interact with firewalld.
+It can be enabled by setting `firewall_driver` to `firewalld` in `containers.conf`.
+The native firewalld driver offers better integration with firewalld, but presently suffers from several limitations.
+It does not support isolation (the `--opt isolate=` option to `podman network create`.
+It also does not currently support connections to a forwarded port of a container on the same host via public IP, only via the IPv4 localhost IP (127.0.0.1).
```

Comment:
given most users are behind A NAT (for ipv4 at least) `public IP` may be misunderstood to refer to the public internet ip.

Maybe the wording should be reversed,i.e.: 

> When initiating a connection from the host system to the forwarded port it ONLY works when connecting to the IPv4 localhost (127.0.0.1). Using the host ips of other interface will not work. However when the connection comes from an external host it will work.

That raises the question about 127.0.0.0/8 ips. Users can do something like `-p 127.0.0.10:8080:8080` to bind another localhost ip? Does that still work or is only 127.0.0.1 special?